code/game/objects/items/weapons/material/misc.dm

```diff
@@ -89,16 +89,20 @@
 	matter = list(/decl/material/solid/plastic = MATTER_AMOUNT_REINFORCEMENT)
 
 /obj/item/hatchet/machete/Initialize()
-	icon_state = "machete[pick("","_red","_blue", "_black", "_olive")]"
+	var/list/machete_icons = list(
```

Comment:
maybe make it static on type? Unsure about instancizing it on init for every machet, tho i guess doesn't matter much.